hpx/parallel/executors/executor_traits.hpp

```diff
@@ -312,6 +312,29 @@ namespace hpx { namespace parallel { HPX_INLINE_NAMESPACE(v3)
         {
             return os_thread_count_helper::call(0, exec);
         }
+
+        ///////////////////////////////////////////////////////////////////////
+        struct has_pending_closures_helper
+        {
+            template <typename Executor>
+            static auto call(wrap_int, Executor& exec) -> bool
+            {
+                return hpx::get_os_thread_count();
```

Comment:
Are the pending closures really the number of OS threads? wouldn't this rather return `false` for stateless executors?